posthog/clickhouse/client/execute_async.py

```diff
@@ -136,14 +139,23 @@ def enqueue_process_query_task(
     query_status = QueryStatus(id=query_id, team_id=team_id, start_time=datetime.datetime.now(datetime.timezone.utc))
```

Comment:
Alternative idea: We put the user ID in the query status here, and thereby have it in the task, without the need to pass it to the task as args. As further advantages, it's then also persisted and returned via API.

Reply:
Hmm, that could work too. Though I don't want to spend time reworking this right now to this degree, if the current solution works. 😄 It doesn't seem super useful _right now_ to return the user ID via the API. I guess if that changes, it might be fine to adjust this